how-to-handle-history-mode-files.md

```diff
@@ -0,0 +1,220 @@
+#### What is History Mode
+
+History mode allows us to capture every version of each record processed by the fivetran connectors.
+In order to keep all versions of the record, we have introduced three new system columns for tables with history mode enabled.
+
+
+Column | Type | Description
+--- | --- | ---
+_fivetran_active | Boolean | TRUE if it is the currently active record. FALSE if it is a historical version of the record. Only one version of the record can be TRUE.
+_fivatran_start | TimeStamp | The time when the record was first created or modified in the source.
```

Comment:
```suggestion
_fivetran_start | TimeStamp | The time when the record was first created or modified in the source.
```
Should n't be TIMESTAMP?

Reply:
@5tran-alexil  It seems fine to me. Let me know if TimeStamp looks odd to you, if yes then I can make changes for DataTypes at all the places